modules/FindPKCS11.cmake

```diff
@@ -8,6 +8,8 @@ if(APPLE)
 	find_library(PKCS11_MODULE NAMES opensc-pkcs11.so HINTS /Library/OpenSC/lib)
 elseif(WIN32)
 	set(PKCS11_MODULE opensc-pkcs11.dll)
+elseif( $ENV{PKCS11_MODULE_PATH} )
+	set( PKCS11_MODULE $ENV{PKCS11_MODULE_PATH} )
```

Comment:
Can you use same code format as rest of file

Reply:
Sure, I've removed the spaces if that's what you meant.